stochastic_physics/sumfln_stochy.f

```diff
@@ -122,12 +122,13 @@ subroutine sumfln_stochy(flnev,flnod,lat1s,plnev,plnod,
 !
 !           compute the sum of the even real      terms for each level
 !           compute the sum of the even imaginary terms for each level
-!
-!           call dgemm('t','n',latl2-lat1+1, 2*(nvar_2-nvar_1+1),
-!     &                 (jcap+2-l)/2,cons1,     !constant
-!     &                 plnev(indev,lat1), len_trio_ls,
-!     &                 flnev(indev,2*nvar_1-1),len_trio_ls,cons0,
-!     &                 apev(2*nvar_1-1,lat1),latl2)
+#ifdef NATIVE_DGEMM
```

Comment:
Why this change?

Reply:
I was testing to see if performance improved using more optimized math libraries (like Intel's MKL library) for resolving the matrix multiplication calls.  By default, ESMF builds the default DGEMM function from the netlib BLAS library which works everything but has little platform optimization.
The default build configuration is to use the ESMF GEMM call (as normal).  I added the cpp pragma as I got tired of commenting/uncommenting the same blocks of code.

Reply:
reverting it back to original form